std/file.d

```diff
@@ -1782,6 +1784,50 @@ if (isInputRange!R && !isInfinite!R && isSomeChar!(ElementEncodingType!R))
     assert(target.timeLastModified(SysTime.min) >= source.timeLastModified);
 }
 
+version(StdDdoc)
+{
+    /++
+     $(BLUE This function is Posix-Only.)
+
+     Returns the time that the given file was last modified.
+     Params:
+        statbuf = stat_t retrieved from file.
+     +/
+    SysTime timeLastModified(ref stat_t statbuf) pure nothrow;
+    /++
+     $(BLUE This function is Posix-Only.)
+
+     Returns the time that the given file was last accessed.
+     Params:
+        statbuf = stat_t retrieved from file.
+     +/
+    SysTime timeLastAccessed(ref stat_t statbuf) pure nothrow;
+    /++
+     $(BLUE This function is Posix-Only.)
+
+     Returns the time that the given file was last changed.
+     Params:
+        statbuf = stat_t retrieved from file.
+     +/
+    SysTime timeStatusChanged(ref stat_t statbuf) pure nothrow;
+}
+else
+version(Posix)
```

Comment:
I know this is nitpicky, but please put these two lines on one line. This is like doing

    else
    if(cond)

which is a bit weird and just wastes a line.